### ISSUE TITLE
Change name of parameters

### ISSUE THREAD

**Issue by DanielAndreasen** [chriskiehl/Gooey#122]
I have something like this in my code:

    parser.add_argument('--spectralType', help='Input spectral type (optional)', default=False)

But the parameter `spectralType` is two words: `spectral` and `type`. Is it possible for Gooey to write two or more words in the title for a given parameter if I specify the `dest` like this:

    parser.add_argument('--spectral type', help='Input spectral type (optional)', default=False, dest='spectralType')

Or maybe in another way?

Thanks for the awesome software :+1:
Daniel

**Comment by chriskiehl** [comment]
Implemented! But.. with a few caveats.

tl'dr version: You can use argparse's `metavar` option to supply a friendly name, but you must use GooeyParser for it to work. This is due to a bug in Argparse.

	from gooey import GooeyParser

	@Gooey(program_name="Metavar Demo")
	def main():
	  my_cool_parser = GooeyParser(description="Using Metavar")
	  my_cool_parser.add_argument("spectralType", metavar="Spectral Type")
	  ...


---

Long version : )

This actually ended up being really interesting. Rather than implement my own terrible logic that tries to correctly split `spectralType` or `spectral_type`, or any of the permutations therein, I opted to make use argparse's `metavar` option. This lets you give a human friendly name to an option. 

e.g. 

    parser.add_argument('--spectralType', help='Input spectral type (optional)', default=False, metavar='Spectral Type')

However, this actually led me to a bug in the actual Argparse code itself! You cannot use the `metavar` option with any of the `StoreConst` subclasses (along with a few others) due to the implementer forgetting to pass the field up to the parent class in the `__init__`. Then, thanks to some funniness in the validation, this leads to an explosion because the subclass then has a `kwargs` option that the parent doesn't know about. 

To get around this, GooeyParser pops the `metavar` option off of the incoming `action`, passes it off to the `__init__` as usual, then finally `setattr`s the `metavar` back onto the object manually thus getting past the weird validation problem. 

So, in short, in order to use ArgumentParser options, you'll have to use GooeyParser. I'll submit a patch to the Python source. But, I'm not totally sure if they're even still doing bugfix releases for 2.7.x : (

It's fun running across bugs in the standard lib. Makes me feel not so bad about my own code being littered with bugs!

**Comment by DanielAndreasen** [comment]
It didn't work. Then I realized I didn't update. Works flawlessly! :beers:  
This project is amazing! It builds a giant bridge between developers and users for small scripts. Thank you very much.

**Comment by chriskiehl** [comment]
Hooray! Glad you're getting some use outta the project! : )
